gitops/ckcp/base/deployment.yaml

```diff
@@ -16,8 +16,19 @@ spec:
     spec:
       serviceAccountName: anyuid
```

Comment:
Why anyuid? Is there a requirement for that?

Reply:
[Here](https://github.com/kcp-dev/kcp/blob/main/manifest/kcp.yaml#L87) no specific Service Account is configured. The syncer needs however to be able to create namespaces.
Generally I see two approaches and we could have both:
- run everything on a development machine and for that kind & kcp binaries (with etcd) should be enough. kcp could possible be run in a container with podman or docker but it is not a must
- run everything on a kubernetes cluster. I think that it is what this [directory](https://github.com/kcp-dev/kcp/tree/main/manifest) provides. We could leverage it, pin a release version and see what else needs to be done

Reply:
@fgiloux If I remember correctly, we had to add the anyuid service account for the deployment because it needed permission to create files
    - 'go build' create kcp executable.
    - .kcp/admin.kubeconfig was created after running 'go run'.

And all this was in our own pod without the kcp image. But now, as you have pointed out, when we move to using the kcp public image, we probably would not need anyuid. We should of course test this out.